skyportal/models/gcn.py

```diff
@@ -50,7 +50,7 @@
     f':{cfg["server.port"]}' if cfg['server.port'] not in [80, 443] else ''
 )
 
-cache_dir = "cache/reports"
+cache_dir = "cache/public_pages/reports"
```

Comment:
In a following PR, we'll want to move the GCN public pages to the public_pages folder too. Just something to keep in mind